frontend/cypress/support/index.js

```diff
@@ -1,3 +1,6 @@
 import 'cypress-fail-fast'
+import installLogsCollector from 'cypress-terminal-report/src/installLogsCollector.js'
+
+installLogsCollector()
 
 import './commands'
```

Comment:
I think eslint will complain about the import at line 6, keep the imports at top